secp256k1-sys/src/lib.rs

```diff
@@ -601,6 +638,38 @@ extern "C" {
                                  output_pubkey: *mut PublicKey,
                                  keypair: *const KeyPair)
                                  -> c_int;
+    // Elligator Swift
+    #[cfg_attr(not(rust_secp_no_symbol_renaming), link_name = "rustsecp256k1_v0_8_1_ellswift_encode")]
+    pub fn secp256k1_ellswift_encode(
+        ctx: *const Context,
+        ell64: *const c_uchar,
+        pubkey: *const PublicKey,
+        rnd32: *const c_uchar,
+    ) -> c_int;
+    #[cfg_attr(not(rust_secp_no_symbol_renaming), link_name = "rustsecp256k1_v0_8_1_ellswift_decode")]
+    pub fn secp256k1_ellswift_decode(
+        ctx: *const Context,
+        pubkey: *mut u8,
+        ell64: *const c_uchar,
+    ) -> c_int;
+    #[cfg_attr(not(rust_secp_no_symbol_renaming), link_name = "rustsecp256k1_v0_8_1_ellswift_create")]
+    pub fn secp256k1_ellswift_create(
+        ctx: *const Context,
+        ell64: *mut c_uchar,
+        seckey32: *const c_uchar,
+        aux_rand32: *const c_uchar,
+    ) -> c_int;
+    #[cfg_attr(not(rust_secp_no_symbol_renaming), link_name = "rustsecp256k1_v0_8_1_ellswift_xdh")]
+    pub fn secp256k1_ellswift_xdh(
+        ctx: *const Context,
+        output: *mut c_uchar,
+        ell_a64: *const c_uchar,
+        ell_b64: *const c_uchar,
+        seckey32: *const c_uchar,
+        party: c_int,
+        hashfp: EllswiftECDHHashFn,
+        data: *const u8,
```

Comment:
`data` is optional in `rustsecp256k1_v0_8_1_ellswift_xdh`. How do we do it in rust bindings? (Similar to previous review comment)